### DISCUSSION TITLE
feature req for cloud ide

### DISCUSSION THREAD

**Discussion by mamatucci** [dbt-labs/docs.getdbt.com#2417] [Archive]
new cloud ide came up recently
https://www.getdbt.com/blog/new-improved-cloud-ide/
I started using it at work and I see a few things wish/need to be  improved

**Comment by obar1** [comment]
## missing vertical split

> it was nice  in the old ide to have both vertical so I can change macros and see the compiled version

![image](https://user-images.githubusercontent.com/387386/202666474-504d00a3-9799-4ea7-ad6a-31a3ddcf0356.png)

**Comment by obar1** [comment]
##  ux to improve

- you have one job 
-  you run it
you click there and nothing happen

![image](https://user-images.githubusercontent.com/387386/202667145-60ae9951-bf7d-4bc7-a2eb-81fd7ddd80a3.png)

- you run another one  and then click works and details are copied as expected
![image](https://user-images.githubusercontent.com/387386/202667420-49d0560b-382b-40b1-b0d5-c1386a9cc2b9.png)

**Comment by obar1** [comment]
## cannot use scratch pad when a job is running

- preview is grayed out not possible to do anything 

![image](https://user-images.githubusercontent.com/387386/202668095-7f7eae76-f864-43fe-9206-520bfea606b9.png)

**Comment by obar1** [comment]
## no way so far to cleanup  your local git workspace 

- you have `feature/1' branch 
- you merge and drop it from git repo
- you have `feature/2' branch 
- you create a new `feature/1' branch in git
- you checkout `feature/1' branch in the ide

you ran is some issues
likely local cloud branch is the 1st - so no branch pruning happening in the cloud :)  ...

**Comment by obar1** [comment]
## you cannot revert just a file 😱 

![image](https://user-images.githubusercontent.com/387386/202675255-d9c070f8-e219-4c5f-8e8b-7a5b27bf6d15.png)

**Comment by obar1** [comment]
## if you delete a file  you dont see it in the UX

![image](https://user-images.githubusercontent.com/387386/202675514-59f1dc3a-ae10-4fc4-ace9-eacdbc7e8271.png)

hit revert - it's back

![image](https://user-images.githubusercontent.com/387386/202675586-5fbe5df2-a49c-4d1f-9889-a86e6ac9007d.png)
